Source/System.Management/Pash/ParserIntrinsics/AstBuilder.cs

```diff
@@ -2271,6 +2271,31 @@ CommandParameterAst BuildCommandParameterAst(ParseTreeNode parseTreeNode)
                                            new ScriptExtent(parseTreeNode), requiresArgument);
         }
 
+        IEnumerable<CommandElementAst> PostprocessCommandElements(IEnumerable<CommandElementAst> elements)
+        {
+            var queue = new Queue<CommandElementAst>(elements);
+            while (queue.Count > 0)
```

Comment:
Last time I was checking `.Any()` was about 100 times slower than `.Count > 0`, so I've come up with general rule of using collection-specific methods when they are available.

It is almost never a bottleneck (even being 100 times slower, it's relatively fast operation indeed), but that's the way I used to write my code. I have no strong implication to write it with or without LINQ (so without problem I may change my style to match the project I am working at), but generally I tend to use `.Count` on the collections that have this property.